src/ProjectTemplates/Web.ProjectTemplates/content/SignalR-CSharp/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "name": {
+    "text": "Empty",
+    "package": "{4f3766dd-5201-42c0-aa83-41e71f56c7a0}",
+    "id": "1011"
```

Comment:
You need to get unique localization IDs for these. @phenning ?

Reply:
Use 1033 for the name id, and 1034 for the description id.

Reply:
Also, the package guid for both of these should be {0CD94836-1526-4E85-87D3-FB5274C5AFC9}  This corresponds with the VSPackage with contains the string resources (in this case - the Template Engine host package for VS)

Reply:
One other thing. I'll take the strings you have in here now and update the resources in our VS repo so we can get a headstart on localization. If the name or description you want displayed in the One ASP.NET dialog changes, please notify me ASAP so we can update the strings for localization. I'll also CC you on the PR for the addition of the resources.

Reply:
thanks all! i'll make the changes you suggested today. thanks all.